### ISSUE TITLE
bad UX for default settings of concurrency

### ISSUE THREAD

**Issue by yureckey** [windmill-labs/windmill#1897]
### Describe the bug

problems with concurrency UI/UX:

add script (typescript) -> metadata -> 
it looks like it is set by default to
Maximum number of runs 5 and Per time window (seconds) 60, whereas it is actually a placeholder only and values not really set. Thus giving user false impression of how it would behave

### To reproduce

I created http native script (I keep autosuggested code), set concurrency "1" and kept 60 seconds as is (I thought it is 60 seconds). Called sync webhooks for this script, and checked the runs - it's log immediately was being bombarded by "postponed" messages several times per second. Then, even though time window I set was passed - script wasn't executed! I've waited couple of minutes but it was still the same, so I cancelled execution.
I've set script concurrency 2 per 60 seconds and called it by sync webhook - it got executed fine, I called webhook several times, one after another without much delay - all got executed without delay.
All this because it didn't occurred to me that time window wasn't actually set. Only after I changed it to some number execution started behave as I expected.

### Expected behavior

default to actually Maximum number of runs 5 and Per time window (seconds) 60, or stated that default is not set

**Comment by rubenfiszel** [comment]
We now do not show placeholders for the concurrency which makes this less confusing

**Comment by yureckey** [comment]
Great! What about validation? Shouldn't it decline negative numbers and zero?
